xtask/src/commands/test.rs

```diff
@@ -1,13 +1,14 @@
 use anyhow::{ensure, Result};
+use std::{thread::sleep, time::Duration};
 use structopt::StructOpt;
 use xtask::*;
 
 const FEATURE_SETS: &[&[&str]] = &[
-    &[],
-    &["otlp-http"],
-    &["otlp-tonic"],
     &["otlp-tonic", "tls"],
+    &["otlp-tonic"],
+    &["otlp-http"],
     &["otlp-grpcio"],
+    &[],
 ];
```

Comment:
Did reordering changed the build time???

Reply:
We're now spawn() ing a test build while we wait for npm i and npm run start. Given it's running in background, we want to start with the biggest set of features. Plus for example the second build is almost free, given it's a subset of the first one.

Reply:
It's not really the biggest set of features... it's 3 different equal paths (except for tonic+tls). I doubt this changes anything at all...

Reply:
as you can see here:

- [xtask run with the new order](https://app.circleci.com/pipelines/github/apollographql/router/321/workflows/4bb4e523-9a7c-4102-986b-7f9065e45534/jobs/1122/parallel-runs/0/steps/0-114)
- [xtask run with the previous order](https://app.circleci.com/pipelines/github/apollographql/router/322/workflows/7cfad089-4589-49d3-8ac7-e25b11bb53ef/jobs/1125/parallel-runs/0/steps/0-114)

this test step went from 3m19s to 2m50s, which is quite the speedup. I do not think there's a huge difference between starting with otpl-http vs otpl-tonic, but there is quite a difference between starting without any features and otpl-tonic and tls.
Given the first build runs in the background, we want to start with the most features.

Reply:
Alright I stand corrected